README.md

```diff
@@ -22,9 +22,9 @@ TiDB ("Ti" stands for Titanium) is an open-source NewSQL database that supports
 
     TiDB acts like it is a MySQL 5.7 server to your applications. You can continue to use all of the existing MySQL client libraries, and in many cases, you will not need to change a single line of code in your application. Because TiDB is built from scratch, not a MySQL fork, please check out the list of [known compatibility differences](https://docs.pingcap.com/tidb/stable/mysql-compatibility).
 
-- __Distributed Transactions with Strong Consistency__
+- __Distributed Transactions__
 
-    TiDB internally shards table into small range-based chunks that we refer to as "Regions". Each Region defaults to approximately 100 MiB in size, and TiDB uses a Two-phase commit internally to ensure that Regions are maintained in a transactionally consistent way.
+    TiDB internally shards table into small range-based chunks that we refer to as "Regions". Each Region defaults to approximately 100 MiB in size, and TiDB uses [async commit](https://pingcap.com/blog/async-commit-the-accelerator-for-transaction-commit-in-tidb-5.0) to ensure that Regions are maintained in a transactionally consistent way.
```

Comment:
The intention of `async commit` isn't to `ensure that Regions are maintained in a transactionally consistent way`. So I think the causal relationship is wrong after modifying it.

Reply:
'Async Commit' is still a two-phase protocol with 'prewrite' and 'commit'. 

Since v5.0 the one-phase commit may be applied as an optimization but it is not common, so I think it's OK to keep 'two-phase' commit unkept here.

Reply:
How about using the language "uses **an [optimized](https://pingcap.com/blog/async-commit-the-accelerator-for-transaction-commit-in-tidb-5.0)** Two-phase commit ~internally~ to ensure that .."

Reply:
I've changed it to "an optimized Two-phase commit".